### ISSUE TITLE
health bar title 

### ISSUE THREAD

**Issue by Paphy435** [Auroratide/poke5e#80]
When you change the name it should also change the name over the health bar 
![Uploading image.jpg…]()

**Comment by Paphy435** [comment]
<img width="594" alt="Screen Shot 2024-06-25 at 7 31 02 PM" src="https://github.com/Auroratide/poke5e/assets/169730797/a930bed7-827b-4de4-831c-fe5d46e4ec45">

**Comment by Paphy435** [comment]
Or maybe it could be its own separate title so you could add a nickname but keep the main name.

**Comment by Auroratide** [comment]
Yeah, the original intent was for the title in the black part to be the pokemon's nickname, and for the part above the health bar to be its species, both so we didn't duplicate information and so the pokemon's species name is shown somewhere.

I can see the reasoning in making it match the pokemon list and instead making the black part the species name, but I wouldn't want to render the pokemon's nickname twice.

![arrow pointing between a pokemon in a list and its open card, showing how it says one thing in the list and something else on the card](https://github.com/Auroratide/poke5e/assets/17010930/c87baa97-5e5b-40bc-98e8-4a230f7f0d38)

**Comment by Paphy435** [comment]
Hmm that makes sense but I’m homebrewing a Pokémon at the moment it’s a pre evo to silvally 
And I don’t want the spiecies name there or I want it to be able to be changed. Also I wanna thank you for doin all this it is a super useful app and I’ve been using it a lot thanks tons :)
